pkg/server/register.go

```diff
@@ -62,5 +62,20 @@ func (s *EkadashiServer) handleLogin(w http.ResponseWriter, r *http.Request) {
 		jsonError(w, http.StatusUnauthorized, fmt.Errorf("incorrect username or password"))
 		return
 	}
+	cookieHash := generateCookieHash()
+	expires := time.Now()
+	cookie := http.Cookie{
```

Comment:
In general ok here, but as a style preference I think you should create session object first, insert it into mongo and then form cookie based on session object